src/app/pending-scans/pending-solution-modal/pending-solution-modal.component.ts

```diff
@@ -76,7 +78,7 @@ export class PendingSolutionModalComponent implements OnInit {
   uploadSolution() {
     this.spinner.show();
     const photosToUpload = this.croppedPages.filter(s => s);
-    this.uploadService.updateSolutionFromPendingScan(this.question, this.solution, photosToUpload).then(() => {
+    this.uploadService.updateSolutionFromPendingScan(console.log, this.question, this.solution, photosToUpload).then(() => {
```

Comment:
it this console.log going to stay in production?

Reply:
Maybe should be replaced with a void function